packages/react/README.md

```diff
@@ -1,8 +1,8 @@
 # Builder.io React SDK
 
-NOTE: If you want to register your React Server Components as custom components in Builder, you will need to use our experimental NextJS SDK [here](../sdks/output/nextjs).
+NOTE: If you want to register your React Server Components as custom components in Builder, you will need to use our experimental NextJS SDK [here](https://github.com/BuilderIO/builder/tree/main/packages/sdks/output/nextjs).
```

Comment:
I think you only need to replace the ../ with packages/

The rest is not needed

Reply:
That would result in below URL which seems incorrect
NG: https://github.com/BuilderIO/builder/blob/packages/sdks/output/nextjs
OK: https://github.com/BuilderIO/builder/tree/main/packages/sdks/output/nextjs

Reply:
I am checking if adding main/packages/ , would work for all.
OK: https://github.com/BuilderIO/builder/blob/main/packages/sdks/output/nextjs

Reply:
Adding main/packages/ would work for some. 
I see some absolute URLs used before.

Reply:
sorry those are broken. You might need to add a missing `/` to the start. Right now the generated URL is:

https://github.com/BuilderIO/builder/blob/om-dev/packages/react/packages/sdks/output/react

Reply:
I'm sorry for confusing you. I think the original absolute URL is the way to go ultimately. My suggestion was off! it won't work properly in npm.

Reply:
No worries, I was unsure myself of whether or not the links would work on both Github and npm. 
I will revert back to absolute URLs.

Reply:
Reverted back to absolute URLs.